src/openforms/forms/models/form_definition.py

```diff
@@ -72,6 +72,13 @@ class FormDefinition(models.Model):
         help_text=_("The total number of Formio components used in the configuration"),
     )
 
+    component_translations = models.JSONField(
+        verbose_name=_("Component translations"),
+        help_text=_("Translations for literals used in components"),
+        blank=True,
+        null=True,
```

Comment:
maybe `default=dict` is a better approach to guarantee there's always a dict datastructure returned from there - then the user only needs to check if the language key is available instead of also checking for null